load/LoadOrganizations.py

```diff
@@ -96,15 +96,72 @@ def hasDamIds(self, languageRecord, typeCode):
 #			if not name in copyrightSet:
 #				resultSet.add(name)
 
+	def process_other_agreement(self, language_record, type_code, fileset_id):
+		licensors = []
+		if language_record.Has_Ambiguous_Methodology():
+			print("ERROR Agreement Type is 'Other', ambiguous methodology and it needs to be fixed for fileset id: %s" % (fileset_id))
+			return None
+
+		if language_record.HasTraditionalRecording() and type_code == "text":
+			if language_record.Licensor() != None:
+				licensors.append(language_record.Licensor())
+			else:
+				print(f"ERROR Agreement Type is 'Other' and Methodology 'TraditionalRecording' but licensor is empty for fileset id: {fileset_id}")
+			return licensors
```

Comment:
We will need to modify this a bit
if hasTraditional
   if type = text
      -- processing as you have
   else 
      -- log "Agreement Type is Other, Methodology is TraditionalRecording, but type is not text. Unable to assign"

similarly for each of the Methodologies. 
I may get feedback on how to assign these, but not sure

Reply:
@bradflood , we have changed the logic for:

- Render. see: https://github.com/faithcomesbyhearing/dbp-etl/pull/135/commits/af2770c3dc402b421a7c15869bb86557ae78a3c3
- TraditionalRecording
- VirtualRecording
- Joint. see https://github.com/faithcomesbyhearing/dbp-etl/pull/135/commits/67ea8cadf9ac33f5732edee56df2c34548a70eae